src/Collections-Support/WideCharacterSet.class.st

```diff
@@ -36,15 +41,24 @@ WideCharacterSet class >> newFrom: aCollection [
 
 { #category : #comparing }
 WideCharacterSet >> = anObject [
-	^self species == anObject species and: [
-		self wideCharacterMap = anObject wideCharacterMap ]
+	self class == anObject class
+		ifTrue: [
+			^ (self byteArrayMap = anObject byteArrayMap) and: [ 
+				self wideCharacterMap = anObject wideCharacterMap ] ].
+	((anObject respondsTo: #size) and: [ anObject respondsTo: #includes: ])
```

Comment:
Beware, a CharacterSetComplement respondsTo: #size... with shouldNotImplement
(until we fix an upper bound to the set of characters)